distributed/diagnostics/tests/test_graph_layout.py

```diff
@@ -102,11 +102,11 @@ async def test_unique_positions(c, s, a, b):
 
 
 @gen_cluster(client=True)
-async def test_layout_with_priorities(c, s, a, b):
```

Comment:
Has this test been removed intentionally? (I'm just wondering because the description says this adds a test, not that it replaces one.)

Reply:
That's intentional. In the initial PR, I added a test with priorities, but I don't think it actually triggered the error case.